include/language.php

```diff
@@ -345,6 +345,24 @@ public function __get( $property ) {
 			return $this->term_props[ $term_prop_type ][ $term_prop ];
 		}
 
+		if ( 'search_url' === $property || 'home_url' === $property ) {
+			/** This filter is documented in wordpress/wp-includes/functions.php */
+			if ( WP_DEBUG && apply_filters( 'deprecated_function_trigger_error', true ) ) {
+				trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+					sprintf(
+						"Class property %1\$s::\$%2\$s is deprecated, use %1\$s::get_%2\$s() instead.\nError handler",
+						esc_html( get_class( $this ) ),
+						esc_html( $property ),
+					),
+					E_USER_DEPRECATED
+				);
+			}
+
+			$property_name = '_' . $property;
+
+			return $this->$property_name;
```

Comment:
Maybe we should use the getter functions instead of the properties directly.

Reply:
Indeed, done in 50f0a4f.